pwa/src/templates/objectDetailTemplate/ObjectDetailTemplate.tsx

```diff
@@ -70,12 +71,65 @@ export const ObjectDetailTemplate: React.FC<ObjectDetailTemplateProps> = ({ obje
-            <Tab className={styles.tab} label={t("Sync")} value={1} />
-            <Tab className={styles.tab} label={t("Object")} value={2} />
+            <Tab className={styles.tab} label={t("Metadata")} value={0} />
+            <Tab className={styles.tab} label={t("Logs")} value={1} />
+            <Tab className={styles.tab} label={t("Sync")} value={2} />
+            <Tab className={styles.tab} label={t("Object")} value={3} />
           </Tabs>
 
           <TabPanel className={styles.tabPanel} value="0">
+            {getObject.isSuccess && (
+              <Table>
+                <TableHead>
+                  <TableRow>
+                    <TableHeader>Id</TableHeader>
+                    <TableHeader>Date created</TableHeader>
+                    <TableHeader>Date modified</TableHeader>
+                    <TableHeader>Owner</TableHeader>
+                    <TableHeader>Organisation</TableHeader>
+                    <TableHeader>Application</TableHeader>
+                    <TableHeader>Schema</TableHeader>
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  <TableRow key={getObject.data.id}>
+                    <TableCell>{getObject.data._self?.id}</TableCell>
+                    <TableCell className={styles.dateContent}>
+                      {formatDateTime(t(i18n.language), getObject.data._self?.dateCreated) ?? "-"}
+                    </TableCell>
+                    <TableCell className={styles.dateContent}>
+                      {formatDateTime(t(i18n.language), getObject.data._self?.dateModified) ?? "-"}
+                    </TableCell>
+                    <TableCell>
+                      <ToolTip tooltip={getObject.data._self?.owner.name ?? ""}>
+                        {getObject.data._self?.owner.id ?? "-"}
+                      </ToolTip>
+                    </TableCell>
+                    <TableCell>
+                      <ToolTip tooltip={getObject.data._self?.organization.name ?? ""}>
+                        {getObject.data._self?.organization.id ?? "-"}
+                      </ToolTip>
+                    </TableCell>
+                    <TableCell>
+                      <ToolTip tooltip={getObject.data._self?.organization.name ?? ""}>
+                        {getObject.data._self?.application.id ?? "-"}
+                      </ToolTip>
+                    </TableCell>
+                    <TableCell>
+                      <span onClick={() => navigate(`/schemas/${getObject.data._self?.schema.id}`)}>
+                        <Link icon={<FontAwesomeIcon icon={faArrowRight} />} iconAlign="start">
+                          {getObject.data._self?.schema.name ?? "-"}
+                        </Link>
+                      </span>
+                    </TableCell>
+                  </TableRow>
+                </TableBody>
+              </Table>
+            )}
```

Comment:
I think a vertical table would be better suited here (there's no way that this table would ever have a second row 👍🏼)